bin/obj2gltf.js

```diff
@@ -119,29 +131,16 @@ if (!defined(gltfPath)) {
     gltfPath = path.join(path.dirname(objPath), modelName + extension);
 }
 
-var options = {
-    binary : argv.binary,
-    separate : argv.separate,
-    separateTextures : argv.separateTextures,
-    compress : argv.compress,
-    optimize : argv.optimize,
-    optimizeForCesium : argv.optimizeForCesium,
-    generateNormals : argv.generateNormals,
-    ao : argv.ao,
-    kmc : argv.kmc,
-    bypassPipeline : argv.bypassPipeline,
-    checkTransparency : argv.checkTransparency,
-    secure : argv.secure,
-    inputUpAxis : argv.inputUpAxis,
-    outputUpAxis : argv.outputUpAxis
-};
-
 console.time('Total');
 
-obj2gltf(objPath, gltfPath, options)
-    .then(function() {
-        console.timeEnd('Total');
-    })
-    .catch(function(error) {
-        console.log(error.message);
-    });
+try {
+    obj2gltf(objPath, gltfPath, argv)
+        .then(function() {
+            console.timeEnd('Total');
+        })
+        .catch(function(error) {
+            console.log(error.message);
```

Comment:
They're errors from bad input parameters instead of data, and mostly for parameters that seem like "advanced use" parameters designed for programmatic calling instead of  use from the CLI.

It seems like if something like, say, an online GUI for hooking up texture files calls this, it should get these errors back as `Promise` rejections instead of `throw`s.